src/push-state-tree.js

```diff
@@ -682,30 +706,28 @@ function preProcessUriBeforeExecuteNativeHistoryMethods(method) {
       }
     }
 
-    root.history[scopeMethod].apply(root.history, args);
+    history[scopeMethod].apply(history, args);
 
     // Chainnable
     return this;
   };
 }
 
 // Wrap history methods
-for (var method in root.history) {
-  if (typeof root.history[method] === 'function') {
+for (var method in history) {
+  if (typeof history[method] === 'function') {
```

Comment:
in this case I want methods from any prototype level, so I think it's ok, since they might change the API in the future we don't know if it will be in the instance level or prototype level...